app/models/contact.rb

```diff
@@ -2,6 +2,7 @@
 
 class Contact < ApplicationRecord
   include PgSearch::Model
+  self.ignored_columns = ['needs_count', 'uncompleted_needs_count', 'completed_needs_count']
```

Comment:
Why do we need this if we've removed the columns?

Reply:
Not necessarily, just good practice. Get it out of here?

Reply:
Done.